docs/apiclient.rst

```diff
@@ -3,6 +3,8 @@ APIClient
 
 FEDn comes with an *APIClient* for interacting with the FEDn network. The APIClient is a Python3 library that can be used to interact with the FEDn network programmatically. 
 
+Connect to the network
+----------------------
 
 The APIClient is available as a Python package on PyPI, and can be installed using pip:
```

Comment:
Confusing, the APIClient is part of the fedn package, which is installed from pip?

Reply:
changed